src/app/core/candidate-management/_store/candidate-management.store.ts

```diff
@@ -4,17 +4,17 @@ import { Injectable } from '@angular/core';
 
 @Injectable()
 export class CandidateManagementStore {
-    private _candidate: BehaviorSubject<any> = new BehaviorSubject(null);
+    private static _candidate: BehaviorSubject<any> = new BehaviorSubject(null);
```

Comment:
two open question for the discussion tomorrow: 
(1) why do we need a subject to keep track of the candidates? 
(2) why is the subject static?